tsconfig.json

```diff
@@ -13,6 +14,7 @@
     "noUnusedParameters": false,
     "noFallthroughCasesInSwitch": true,
     "forceConsistentCasingInFileNames": true,
+    "experimentalDecorators": true
```

Comment:
Is there any reason why you included this? The transformer code doesn't have any decorator, so I think it is not necessary (but it is for the tests 😉 )